### PR TITLE
Improve ced pipeline

### PR DESCRIPTION
**Pull request by CodyCBakerPhD**
@bendichter @luiztauffer @alejoe91 The usual updates to the modern standard for the pipeline conversion, featuring `quick_write` and 'full save' for sorting extractors and lfp recordings from the notebook.

Need to fully test it, which I'll do so here in the next couple hours.

Points to `add_ced_metadata` for readability.

### CODE REVIEW

**Comment by CodyCBakerPhD** [comment]
@bendichter Some small fixes were in order and are dependent on [spikeextractors PR #531](https://github.com/SpikeInterface/spikeextractors/pull/531)

Otherwise running smoothly, will have example files to share for next meeting

File: mease_lab_to_nwb/convert_ced/cedstimulusinterface.py [hunk 1]
```diff
@@ -31,13 +30,16 @@ def check_module(nwbfile, name, description=None):
         return nwbfile.create_processing_module(name, description)
 
 
-def intervals_from_traces(traces):
+def intervals_from_traces(recording: RecordingExtractor):
     """Extract interval times from TTL pulses."""
+    traces = recording.get_traces(channel_ids=[1, 2])
```
Comment (alejoe91) [inline mease_lab_to_nwb/convert_ced/cedstimulusinterface.py:35]: @CodyCBakerPhD why `channel_ids` is always `[1, 2]`? I would add `channel_ids` as an extra arg

File: notebooks/CED_spikeinterface_pipeline.ipynb [hunk 2]
```diff
@@ -104,18 +80,22 @@
...
-    "recording = se.CEDRecordingExtractor(ced_file, smrx_ch_inds=rhd_channels)"
+    "recording = se.CEDRecordingExtractor(ced_file, smrx_channel_ids=rhd_channels)\n",
+    "\n",
+    "prb_channels = [\n",
+    "    42, 36, 32, 38, 39, 47, 45, 37, 44, 46, 48, 35, 34, 40, 41, 43, 62, 60, 58, 56, 54, 52, 50, 33, 49, 51, 53,\n",
+    "    55, 57, 59, 61, 63, 1, 3, 5, 7, 9, 11, 13, 30, 14, 12, 10, 8, 6, 4, 2, 0, 21, 27, 31, 25, 24, 16, 18, 26,\n",
+    "    19, 17, 15, 28, 29, 23, 22, 20\n",
+    "]\n",
+    "prb_locations = [[0, x * 20] for x in range(len(prb_channels))]\n",
+    "recording.set_channel_locations(locations=np.array(prb_locations)[prb_channels, :])"
```
Comment (alejoe91) [inline notebooks/CED_spikeinterface_pipeline.ipynb:91]: We should provide a test file instead. I added it to the repo, but it's not there anymore. I'll make a PR with the cambridge neurotech probe and tetrodes probes

Comment (alejoe91) [reply notebooks/CED_spikeinterface_pipeline.ipynb:91]: Actually, the probe file requested for these kind of CED files is in main: https://github.com/catalystneuro/mease-lab-to-nwb/blob/main/probe_files/cambridge_neurotech_H3.prb

**Comment by CodyCBakerPhD** [comment]
Closing this branch and opening new one per all the changes discussed from the last week, and moving the smaller more immediate changes to a new branch

**Pull request closed without merging**